runner_manager/models/settings.py

```diff
@@ -42,7 +42,7 @@ class Settings(BaseSettings):
     ]
     github_webhook_secret: Optional[SecretStr] = None
     log_level: LogLevel = LogLevel.INFO
-    runner_groups: List[BaseRunnerGroup] = []
+    runner_groups: List[RunnerGroup] = []
```

Comment:
Why?

Reply:
I made a mistake I thihn `runner_group` is a list of RunnerGroup and not BaseRunnerGroup. Maybe I am wrong.

Reply:
BaseRunnerGroup might not be needed anymore, I believe it was originally created to deal with the circular dependency issue. But while it exists, I think it's best we set runner_groups as `BaseRunnerGroup`

Reply:
ok